src/main/java/org/group23/NumericalQuestion.java

```diff
@@ -0,0 +1,36 @@
+package org.group23;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.ElementCollection;
+import jakarta.persistence.Entity;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Entity
+public class NumericalQuestion extends Question {
+
+    //single numerical answer with a restricted length
+    @Column
+    private Double numericalAnswer;
```

Comment:
Went and checked TextQuestion: @CamSommer confirm on this please but I believe we want to store _all_ provided answers here as an ElementCollection? We should be consistent however we choose to do it.

Reply:
Yes, it should be a collection of all answers. Otherwise, we would need duplicates of the question every time an answer is submitted.

Reply:
done